src/util.cpp

```diff
@@ -364,7 +362,7 @@ void ParseParameters(int argc, const char* const argv[])
         }
 #ifdef WIN32
         boost::to_lower(str);
-        if (boost::algorithm::starts_with(str, "/"))
+        if (str.size() > 0 && str[0] == '/')
```

Comment:
I agree that not replacing it is urgent, this originally stemmed from a one-off from another patch I was working on.  Due to @fanquake's suggestion I looked at replacing all usages, and found cases where there was more than a single-character prefix/postfix match happening.

I'm dropping this PR altogether, not worth the change.